File: ascii.c
#include<stdio.h>

int main()
{
 int x;
 int k=1;
 
 for(x=0;x<=255;x++)
 {
   printf("\t\t%d------>%c",x,x);
   if(k%5==0)
    printf("\n");
    k++;
	}
printf("\n Capital Alphabet \n");
k=1;
 
 for(x=65;x<=91;x++)
 {
   printf("\t\t%d------->%c",x,x);
   if(k%5==0)
    printf("\n");
    k++;
}


printf("\n Small Alphabet \n");
k=1;
 
 for(x=97;x<=122;x++)
 {
   printf("\t\t%d------->%c",x,x);
   if(k%5==0)
    printf("\n");
    k++;
}



return 0;
	
}

File: coforge_sep_2024/JavaProject1/src/main/java/Ascii.java
public class Ascii {
    public static void main(String[] args) {
        int x = 65;
        System.out.println(x);
        System.out.println((char) x);
        System.out.println((char) (++x));

        char p = 'Z';
        System.out.println(p);
        System.out.println((int) p);
        System.out.println("capital alphabet ");
        x = 64;
        while (x++ <= 89) {
            System.out.print(x + "\t" + (char) x + "\t");
            if (x % 5 == 0)
                System.out.println();
        }

        System.out.println("\n small alphabet ");
        int j=1;
        char g='a';

        while(g!='{'){
            System.out.print((g++)+"\t");
            if(j%5==0)
                System.out.println();
            j++;
        }
    }
    static void smallAlphabet(){
       // x=97 to 122 ,,using while loop and cast char display it

    }
}
